src/GraphCards.ts

```diff
@@ -25,7 +25,7 @@ export class Card {
                 },
             },
             axisX: {
-                title: 'Days',
```

Comment:
@OnkarRuikar can you change **Days (UTC)** back to **Days**. I guess that will be fine.